diff --git a/public/testem/testem_connection.js b/public/testem/testem_connection.js
index 5bee5dee772eaa75d968deaccb9e1e2d98b85cb7..914212647d11ff6e49f38b3d422204c80ccd68e9 100644
--- a/public/testem/testem_connection.js
+++ b/public/testem/testem_connection.js
@@ -175,7 +175,7 @@ function patchEmitterForWildcard(socket) {
 }
 
 function initSocket(id) {
-  socket = io({ reconnectionDelayMax: 1000, randomizationFactor: 0 });
+  socket = io('http://localhost:7357', { reconnectionDelayMax: 1000, randomizationFactor: 0 });
   patchEmitterForWildcard(socket);
 
   socket.emit('browser-login', getBrowserName(navigator.userAgent), id);